content/en/docs/concepts/extend-kubernetes/api-extension/custom-resources.md

```diff
@@ -9,40 +9,49 @@ weight: 20
 
 {{% capture overview %}}
 
-This page explains *custom resources*, which are extensions of the Kubernetes
-API, including when to add a custom resource to your Kubernetes cluster and when
-to use a standalone service. It describes the two methods for adding custom
-resources and how to choose between them.
+*Custom resources* are extensions of the Kubernetes API. This page discusses when to add a custom
+resource to your Kubernetes cluster and when to use a standalone service. It describes the two
+methods for adding custom resources and how to choose between them.
 
 {{% /capture %}}
 
 {{% capture body %}}
 ## Custom resources
 
-A *resource* is an endpoint in the [Kubernetes API](/docs/reference/using-api/api-overview/) that stores a collection of [API objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/) of a certain kind. For example, the built-in *pods* resource contains a collection of Pod objects.
+A *resource* is an endpoint in the [Kubernetes API](/docs/reference/using-api/api-overview/) that stores a collection of
+[API objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/) of a certain kind. For example, the built-in *pods* resource contains a collection of Pod objects.
 
-A *custom resource* is an extension of the Kubernetes API that is not necessarily available on every
-Kubernetes cluster.
-In other words, it represents a customization of a particular Kubernetes installation.
+A *custom resource* is an extension of the Kubernetes API that is not necessarily available in a default
+Kubernetes installation. It represents a customization of a particular Kubernetes installation. However,
+many core Kubernetes functions are now built using custom resources, making Kubernetes more modular.
 
 Custom resources can appear and disappear in a running cluster through dynamic registration,
 and cluster admins can update custom resources independently of the cluster itself.
-Once a custom resource is installed, users can create and access its objects with
-[kubectl](/docs/user-guide/kubectl-overview/), just as they do for built-in resources like *pods*.
+Once a custom resource is installed, users can create and access its objects using the
+[kubectl](/docs/user-guide/kubectl-overview/) command, just as they do for built-in resources like
```

Comment:
Instead of saying "using the kubectl command", I think we should say "using kubectl" - because kubectl is the tool and not the command itself.